gallery.js

```diff
@@ -1,2 +1,30 @@
 import './sass/gallery.scss'
 import '@fortawesome/fontawesome-free/css/all.css'
+
+// Navbar functionality
+var navlinks = document.getElementsByClassName("navlinks");
```

Comment:
Possibly refactor to jQuery or at least reduce the implementation to use less loops